modules/nodes-pods-using-example.adoc

```diff
@@ -36,10 +36,11 @@ spec:
   schedulerName: default-scheduler
   terminationGracePeriodSeconds: 30
   nodeName: ip-10-0-140-16.us-east-2.compute.internal
+  # TODO: How to update this example? Should we just replace it?
```

Comment:
@s-urbaniak @stlaz  This one has securityContext set too, but callout <3> below mentions about it being restrictive. Can we just replace the existing one with what we've been updating, or do we need to do something different here?

Reply:
we should actually move it outside the "default" NS, too. This appears to be some kind of an ancient dump of a pod from the cluster. I don't think pods in the "default" NS would even get SCC annotations.

We should ask the owners  of the doc about what it's supposed to do with this one I guess.

Reply:
Thanks, I'll see who I can find to help with this one

Reply:
@mburke5678 @sunilcio Do you know who could help figure out what we should do with this example?

This PR is in the context of trying to update all of our pod examples to comply with restricted PSA enforcement.

Reply:
This output looks dump of `oc describe pod` and not an an actual pod yaml file. Here we see output like `status`, `hostIP` which are not part of pod yaml. I see this section is intended to describe example pod yaml, I guess we can use pod example file from other section and add here and remove the current one?